pkg/minikube/cluster/cluster.go

```diff
@@ -224,6 +225,12 @@ func createDriverOptions(driver drivers.Driver, explicitOptions map[string]inter
 
 func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	fmt.Println(fmt.Sprintf("Downloading ISO '%s'", m.MinikubeISO))
+	tmpISOFile, err := ioutil.TempFile("", "tmpISOFile")
+	fmt.Println("TMP FILE IS", tmpISOFile.Name())
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpISOFile.Name())
 
 	// store the iso inside the MINISHIFT_HOME dir
 	response, err := http.Get(m.MinikubeISO)
```

Comment:
Maybe we could also append 'sha256' to the URL and check whether there is a checksum. If there is, we can also add checksum verification of the download capturing potential transfer problems.